core/src/main/java/com/datastax/oss/driver/api/core/config/DriverConfigLoader.java

```diff
@@ -58,13 +62,33 @@
    */
   @NonNull
   static DriverConfigLoader fromClasspath(@NonNull String resourceBaseName) {
+    return fromClasspath(resourceBaseName, DRIVER_CLASS_LOADER);
+  }
+
+  /**
+   * Just like {@link #fromClasspath(java.lang.String)} except that the loader will use the provided
```

Comment:
1. I would avoid the word "loader" since we are talking about a config loader that uses a class loader... to avoid confusion I would rephrase to `... except that application configuration resources will be located using the provided {@link ClassLoader} instead of the driver's {@linkplain DriverConfigLoader#DRIVER_CLASS_LOADER default one}`. Just a suggestion, you don't have to agree :-)
2. I wonder if the second paragraph is really necessary, given that this is very hard to explain and, arguably, only of concern to a very small set of users.